pom.xml

```diff
@@ -85,7 +85,7 @@
     <dependency>
       <groupId>com.ning</groupId>
       <artifactId>async-http-client</artifactId>
-      <version>1.7.24</version>
+      <version>1.7.24-jenkins-1-20181114.155326-3</version>
```

Comment:
To be replaced with the release once [async-http-client PR#3](https://github.com/jenkinsci/async-http-client/pull/3) is merged and released.